runtime/src/parachains.rs

```diff
@@ -378,34 +369,21 @@ impl<T: Trait> Module<T> {
 */
 }
 
-impl<T: Trait> ProvideInherent for Module<T> {
-	type Inherent = Vec<AttestedCandidate>;
-	type Call = Call<T>;
+pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"parchain";
```

Comment:
maybe let's make it `newheads` because `parchain` is making my eyes hurt.
docs?